### PR TITLE
feat: cloud provider storage modules outputs for infra revamp

### PR DESCRIPTION
**Pull request by jdongo-aneo**

### CODE REVIEW

**Review by ngruelaneo — APPROVED** [review]

File: storage/aws/mq/outputs.tf [hunk 1]
```diff
@@ -46,3 +46,24 @@ output "engine_type" {
...
+#new Outputs 
+output "env" {
+  description = "Elements to be set as environment variables"
+  value = ({
+    "Components__QueueAdaptorSettings__ClassName"           = var.adapter_class_name
+    "Components__QueueAdaptorSettings__AdapterAbsolutePath" = var.adapter_absolute_path
+    "Amqp__Host"                                            = aws_mq_broker.mq.engine_type == "ActiveMQ" ? trim(split(":", aws_mq_broker.mq.instances[0].endpoints[1])[1], "//") : trim(split(":", aws_mq_broker.mq.instances[0].endpoints[0])[1], "//")
+    "Amqp__Port"                                            = aws_mq_broker.mq.engine_type == "ActiveMQ" ? tonumber(split(":", aws_mq_broker.mq.instances[0].endpoints[1])[2]) : tonumber(split(":", aws_mq_broker.mq.instances[0].endpoints[0])[2])
+    "Amqp__Scheme"                                          = var.scheme
+    "Amqp__CaPath"                                          = "${var.path}/chain.pem"
```
Comment (lemaitre-aneo) [inline storage/aws/mq/outputs.tf:59]: There is no CaPath for AmazonMQ, as the AWS cert is already trusted
```suggestion
```

File: storage/aws/mq/variables.tf [hunk 2]
```diff
@@ -108,3 +108,33 @@ variable "kms_key_id" {
...
+  description = "The scheme for the AMQP"
+  type        = string
+  default     = "AMQPS"
+}
+
+variable "path" {
+  description = "Path for mounting secrets"
+  type        = string
+  default     = "/amqp"
+}
```
Comment (lemaitre-aneo) [inline storage/aws/mq/variables.tf:134]: There is no secret to mount as the AmazonMQ cert is trusted
```suggestion
```

File: storage/gcp/memorystore/redis/variables.tf [hunk 3]
```diff
@@ -145,3 +145,39 @@ variable "customer_managed_key" {
...
+  type        = string
+  default     = "ArmoniK.Adapters.Redis.ObjectStorage"
+}
+
+variable "path" {
+  description = "Path for mounting secrets"
+  type        = string
+  default     = "/redis"
+}
+
```
Comment (lemaitre-aneo) [inline storage/gcp/memorystore/redis/variables.tf:160]: The MemoryStore cert is already trusted, no need to mount the cert.

```suggestion
```

File: storage/gcp/memorystore/redis/outputs.tf [hunk 4]
```diff
@@ -64,3 +64,23 @@ output "region" {
...
+
+#new Outputs 
+output "env" {
+  description = "Elements to be set as environment variables"
+  value = ({
+    "Components__ObjectStorage" = var.object_storage_adapter
+    "Redis__EndpointUrl"        = "${google_redis_instance.cache.read_endpoint}:${google_redis_instance.cache.read_endpoint_port}"
+    "Redis__Ssl"                = var.ssl_option
+    "Redis__ClientName"         = var.client_name
+    "Redis__CaPath"             = "${var.path}/chain.pem"
```
Comment (lemaitre-aneo) [inline storage/gcp/memorystore/redis/outputs.tf:76]: The MemoryStore cert is already trusted, no need to mount the cert.
```suggestion
```

**Review by lemaitre-aneo — APPROVED** [review]

**Pull request merged**